src/features/onboarding/screen/index.tsx

```
@@ -0,0 +1,57 @@
+import { FirstOnboardingContainer } from '../containers/FirstOnboardingContainer';
+
+export default function OnboardingScreen() {
+  const [activeIndex, setActiveIndex] = useState(0);
+  const [showOnboarding, setShowOnboarding] = useState(true);
+  const swiperRef = useRef<SwiperType | null>(null);
+
+  const handleOnboardingComplete = () => {
+    setShowOnboarding(false);
+  };
+
+  const handleNext = () => {
```

ESLint

[error] 'handleNext' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)